<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimingConstraintSliders</class>
 <widget class="QWidget" name="TimingConstraintSliders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>57</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Window Start:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Window End:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSlider" name="startSlider">
           <property name="maximum">
            <number>1800</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="startSpinbox">
           <property name="suffix">
            <string> s</string>
           </property>
           <property name="maximum">
            <number>1800</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSlider" name="endSlider">
           <property name="maximum">
            <number>1800</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="endSpinbox">
           <property name="suffix">
            <string> s</string>
           </property>
           <property name="maximum">
            <number>1800</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QToolButton" name="removeButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources.qrc">
         <normaloff>:/icons/16x16/gui/images/16x16/actions/cancel.png</normaloff>:/icons/16x16/gui/images/16x16/actions/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startSpinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>startSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>endSpinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>endSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>startSpinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>endSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>endSpinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
